File: comparacao_python/Automovel.py
class Automovel:
    def __init__(self,marca="", modelo="", cor="",preco=""):
        self.marca = marca
        self.modelo = modelo
        self.cor = cor 
        self.preco = preco

    def Informa_automovel(self):
        print(f"Marca:  {self.marca}")
        print(f"Modelo:  {self.modelo}")
        print(f"Cor:  {self.cor}")
        print(f"Preço:  {self.preco}")

File: exec/src/mtd_destrutor/Automovel.java
package mtd_destrutor;

public class Automovel {
    private String marca;
    private String modelo;
    private String cor;
    private double preço;

    Automovel(String marca, String modelo, String cor, double preço){
        this.marca = marca;
        this.modelo = modelo;
        this.cor = cor;
        this.preço = preço;

    }

    String dados;

    public String mostra_Automovel(){

        dados = "Marca:\t" + marca + "\n" + "Modelo:\t" + modelo + "\n" + "Cor:\t" + cor + "\n" + "Preço R$" + preço;
        return dados;

    }
}
